langserve/server.py

```diff
@@ -192,18 +208,22 @@ def add_routes(
             "Use `pip install sse_starlette` to install."
         )
 
+    namespace = path or ""
+
+    model_namespace = path.strip("/").replace("/", "_")
```

Comment:
Are there restrictions on pydantic names? 
- if so, there are more characters in paths that we might need to handle
- if not, can we skip this and pass around the `path` instead? Might be more straightforward than a new `namespace` concept

Reply:
Good points. Will check wasn't being methodlogical here